roles/taca/templates/site_fake_delivery.yml.j2

```diff
@@ -0,0 +1,85 @@
+log:
+    file: "{{ ngi_pipeline_site_path }}/log/taca.log"
+
+deliver:
+    rootpath: "/proj/{{ ngi_pipeline_site_delivery }}/nobackup/NGI"
+    analysispath: <ROOTPATH>/ANALYSIS/<PROJECTID>
+    datapath: <ROOTPATH>/DATA/<PROJECTID>
+    stagingpath: <ROOTPATH>/DELIVERY/<PROJECTID>
+    deliverypath: /proj/<UPPNEXID>/INBOX/<PROJECTID>
+    report_aggregate: "ngi_reports ign_aggregate_report -n {{ site_full }} --pandoc_binary"
+    report_sample: "ngi_reports ign_sample_report -n {{ site_full }} --pandoc_binary"
+    reportpath: <ANALYSISPATH>/piper_ngi
+    logpath: <REPORTPATH>/logs
+    deliverystatuspath: <REPORTPATH>/08_misc
+    operator: "{{ recipient_mail }}"
+    hash_algorithm: md5
+    files_to_deliver:
+        -   
+            - <ROOTPATH>/DELIVERY.README.txt
+            - <STAGINGPATH>
+        -
+            - <LOGPATH>/version_report.txt
+            - <STAGINGPATH>/00-Reports
+            - required: True
+        -
+            - <REPORTPATH>/delivery/reports/<PROJECTID>_aggregate_report.csv
+            - <STAGINGPATH>/00-Reports
+            - no_digest: True
+        -
+            - <REPORTPATH>/delivery/reports/<SAMPLEID>_ign_sample_report.html
+            - <STAGINGPATH>/<SAMPLEID>/00-Reports
+            - no_digest_cache: True
+        - 
+            - <ANALYSISPATH>/piper_ngi/07_variant_calls/<SAMPLEID>*.snpEff.summary.*
```

Comment:
unless there is a good reason not to, I think you should include a `.` after `<SAMPLEID>` to make sure that the complete sample id matches (e.g. `Sample1.suffix` should match but not `Sample10.suffix`)